src/components/Assemblies/CabinetRole.svelte

```diff
@@ -1,21 +1,67 @@
 <script lang="ts">
+	import type { AssemblyMember } from '$lib/datasheets/assembly-member';
 	import type { CabinetSeat } from './shared';
 
 	export let cabinets: CabinetSeat[];
 	export let role: CabinetSeat['role'];
+
+	const show = (event: MouseEvent, member: AssemblyMember) => {
+		const memberName = `${member.firstname} ${member.lastname}`;
+		const tooltipElem = document.getElementById('tooltip');
+
+		if (tooltipElem) {
+			tooltipElem.setAttribute(
+				'style',
+				`left: ${event.layerX}px; top: ${event.layerY + 20}px; display: flex;`
+			);
+		}
+
+		const memberElem = document.getElementById('member-name');
+		const roleElem = document.getElementById('role');
+
+		if (memberElem) {
+			memberElem.textContent = memberName;
+		}
+
+		if (roleElem) {
+			roleElem.textContent = member.assemblyRole.role;
+		}
+	};
+
+	const hide = () => {
+		const tooltipElem = document.getElementById('tooltip');
+
+		if (tooltipElem) {
+			tooltipElem.setAttribute('style', `display: none;`);
+		}
```

Comment:
We better avoid using a selector like `getElementById` and then set its attribute or content, when modern front-end libraries (Svelte, Vue, React, etc.) already provide a better way to do it.

**Why?** 
1. Selecting using id is in the global scope. We never know if someone already uses this id somewhere else, especially in a big project.
2. The code editor won't warn you if you misspell the id

Svelte does have [bind:this](https://svelte.dev/docs/element-directives#bind-this) to assign an element into a variable, but in this case you don't need it. I suggest you design the solution this way:
1. You create a `tooltip` variable which can be either an object (containing member name, role, position, etc.) or null. This object represents the tooltip state.
2. Then show and hide the function just "set" the correct data to the `tooltip` variable.
3. On the HTML element, then you can render content/attributes according to the `tooltip` variable. See more about expression in Svelte: [Your First Component](https://learn.svelte.dev/tutorial/your-first-component) and [Dynamic Attribute](https://learn.svelte.dev/tutorial/dynamic-attributes)

By making a template (HTML, SVG, etc.) depend purely on the variable (aka state), make UI a lot more predictable and easier for other people to understand your code.

Reply:
I've fixed according to your suggestion.